### PR TITLE
Added videos, speakers still missing

### PR DESCRIPTION
**Pull request by chriseidhof**
We need to add twitter links for the following speakers:

```
The Organizers of Alt:
  twitter: altconf
Victor Broido:
  twitter: TODO
Josh Michaels:
  twitter: TODO
Thomas Engel:
  twitter: TODO
Philippe Casgrain:
  twitter: TODO
Michael Lopp:
  twitter: TODO
Jay Freeman:
  twitter: saurik
Mattt Thompson:
  twitter: mattt
Jonah Neugass:
  twitter: TODO
Jean MacDonald:
  twitter: TODO
Ouriel Ohayon:
  twitter: TODO
Mikey Ward:
  twitter: TODO
Jeff Kelley:
  twitter: TODO
Sam Marshall:
  twitter: Dirk_Gently
Michael Crump:
  twitter: TODO
Michael Mace:
  twitter: TODO
Andrew Stone:
  twitter: TODO
Yang Meyer:
  twitter: yangmeyer
Carla White:
  twitter: TODO
Nadyne Richmond:
  twitter: TODO
Sofia Dvoynos:
  twitter: TODO
Jay Thrash:
  twitter: jaythrash
Mark Pauley:
  twitter: TODO
Brent Simmons:
  twitter: brentsimmons
Jonathan Rhyne:
  twitter: jdrhyne
```

### CODE REVIEW

**Comment by chriseidhof** [comment]
Anybody who feels like adding those? Otherwise I'll do it tomorrow.

File: Rakefile [hunk 1]
```diff
@@ -7,7 +7,7 @@ require 'yt'
 task :default => [:build, :lint_speakers, :lint_events]
 
 task :fetchyt, [:user] => :dotenv do |t, args|
-  url = "https://www.youtube.com/user/#{args[:user]}/"
+  url = "https://www.youtube.com/channel/UChiwrWoactp8mOs70j53zYw?spfreload=10"
```
Comment (ashfurrow) [inline Rakefile]: Did this get accidentally checked-in?

**Comment by johankj** [comment]
Here’s most of them:

```
The Organizers of Alt:
  twitter: altconference
Philippe Casgrain:
  twitter: philippec
Michael Lopp:
  twitter: rands
Jay Freeman:
  twitter: saurik
Mattt Thompson:
  twitter: mattt
Jonah Neugass:
  twitter: bgntsty
Jean MacDonald:
  twitter: macgenie
Ouriel Ohayon:
  twitter: ourielohayon
Mikey Ward:
  twitter: wookiee
Jeff Kelley:
  twitter: slaunchaman
Sam Marshall:
  twitter: Dirk_Gently
Michael Crump:
  twitter: mbcrump
Michael Mace:
  twitter: michaelmace
Andrew Stone:
  twitter: twittelator
Yang Meyer:
  twitter: yangmeyer
Carla White:
  twitter: carlawhite
Nadyne Richmond:
  twitter: nadyne
Sofia Dvoynos:
  twitter: sofiux
Jay Thrash:
  twitter: jaythrash
Mark Pauley:
  twitter: unsaturated
Brent Simmons:
  twitter: brentsimmons
Jonathan Rhyne:
  twitter: jdrhyne
```

The panels  have "Panel" as the speaker. That should probably be changed as well.

File: Rakefile (hunk 1 above)
Comment (chriseidhof) [inline Rakefile]: Oh yeah, sorry!

**Pull request merged**